docs/ROOTFS_DISCOVERY.md

```diff
@@ -12,11 +12,11 @@ The recommended way to boot a [`systemd`](https://systemd.io/) based
 components:
 
 1. A boot loader,
-   i.e. [`systemd-boot`](https://www.freedesktop.org/software/systemd/man/latest/systemd-boot.html)
+   e.g. [`systemd-boot`](https://www.freedesktop.org/software/systemd/man/latest/systemd-boot.html)
```

Comment:
sorry, but grub is not a recommended way to boot from my pov?

Reply:
There are other bootloaders. But I don't think this matters too much, I'll change it back.